tests/test_project.py

```diff
@@ -2057,24 +2057,17 @@ def test_import_project_rolling(repos_tmpdir):
                           import: true
                       '''})
 
-    cmd(f'init -l {manifest_repo}')
+    cmd(f'change zephyr --revision pull/123/head')
+    cmd(f'change zephyr --path modules/lib')
```

Comment:
Why are you removing existing test code to suit the purpose of this new command instead of just adding more?

These changes are not well explained. This is just not OK. You need to write a good commit message justifying why these changes are coming in instead of just making small modifications and not responding to comments with words as well as code.

For example, this still doesn't work when you get the project from a `self: import:`; I requested this in the original attempt in the zephyr repository, pointed you again to this comment when you tried in the west repo, and it's still not addressed, nor have you asked any questions to clarify if you do not understand why this is important and why your current approach doesn't seem to work.

This effort is not going well; if you want this merged, you will need to start being more communicative instead of just posting new code changes with no explanation.

Reply:
Regarding removing existing code it was by mistake.
I'll add more comments and and I'll make the corrections.